src/care/PluginData.h

```diff
@@ -0,0 +1,65 @@
+#ifndef _CARE_PluginData_H_
+#define _CARE_PluginData_H_
+
+#include "chai/ExecutionSpaces.hpp"
```

Comment:
You'll need to clean up your includes (you are missing headers for std::functional, std::unordered_map, and chai::PointerRecord). Also, CHAICallback.h does not need to be included here (it might be needed in PluginData.cpp, though I haven't checked yet).

Reply:
This hasn't changed.